src/pages/Reports/state/tests/OverviewMapState.unit.test.js

```diff
@@ -88,31 +99,55 @@ describe('OverviewMapState reducer and actions', () => {
           reports: mockedReportsList
         });
       });
```

Comment:
There seems to be something wrong here. These parantheses seem not to match based on indentation and the variables `actualActions` and `actualState`, used below, are not defined.

Reply:
Yup,
https://github.com/FixMyBerlin/fixmy.frontend/pull/529/commits/e2583aacd6697f779f067134e99fda2320584d96
and
https://github.com/FixMyBerlin/fixmy.frontend/pull/529/commits/8a49c12258cfce96c7d946ed5726e0ebb85dad0a
messed things up.
Fixing it.